docs/content/troubleshooting.md

```diff
@@ -1,137 +1,11 @@
-
-After you update the [ConfigMap](/nginx-ingress-controller/configuration/global-configuration/configmap-resource) resource, you can immediately check if the configuration was successfully applied by NGINX:
-```
-$ kubectl describe configmap nginx-config -n nginx-ingress
-Name:         nginx-config
-Namespace:    nginx-ingress
-Labels:       <none>
-. . .
-Events:
-  Type    Reason   Age                From                      Message
-  ----    ------   ----               ----                      -------
-  Normal  Updated  11s (x2 over 26m)  nginx-ingress-controller  Configuration from nginx-ingress/nginx-config was updated
-```
-Note that in the events section, we have a `Normal` event with the `Updated` reason, which informs us that the configuration was successfully applied.
-
-### Checking the Generated Config
-
-For each Ingress/VirtualServer resource, the Ingress Controller generates a corresponding NGINX configuration file in the `/etc/nginx/conf.d` folder. Additionally, the Ingress Controller generates the main configuration file `/etc/nginx/nginx.conf`, which includes all the configurations files from `/etc/nginx/conf.d`. The config of a VirtualServerRoute resource is located in the configuration file of the VirtualServer that references the resource.
-
-You can view the content of the main configuration file by running:
-```
-$ kubectl exec <nginx-ingress-pod> -n nginx-ingress -- cat /etc/nginx/nginx.conf
-```
-
-Similarly, you can view the content of any generated configuration file in the `/etc/nginx/conf.d` folder. 
-
-You can also print all NGINX configuration files together:
-```
-$ kubectl exec <nginx-ingress-pod> -n nginx-ingress -- nginx -T
-```
-However, this command will fail if any of the configuration files is not valid.
-
-### Checking the Live Activity Monitoring Dashboard
-
-The live activity monitoring dashboard shows the real-time information about NGINX Plus and the applications it is load balancing, which is helpful for troubleshooting. To access the dashboard, follow the steps from [here](/nginx-ingress-controller/logging-and-monitoring/status-page).
-
-### Running NGINX in the Debug Mode
-
-Running NGINX in the [debug mode](https://docs.nginx.com/nginx/admin-guide/monitoring/debugging/) allows us to enable its debug logs, which can help to troubleshoot problems in NGINX. Note that it is highly unlikely that a problem you encounter with the Ingress Controller is caused by a bug in the NGINX code, but it is rather caused by NGINX misconfiguration. Thus, this method is rarely needed.
 
-To enable the debug mode, set the `error-log-level` to `debug` in the [ConfigMap](/nginx-ingress-controller/configuration/global-configuration/configmap-resource) and use the `-nginx-debug` [command-line argument](/nginx-ingress-controller/configuration/global-configuration/command-line-arguments) when running the Ingress Controller.
+This doc is now available at https://docs.nginx.com/nginx-ingress-controller/troubleshooting/
```

Comment:
delete this doc and add an alias in the new file's frontmatter

Reply:
This was removed and I added the alias in app-protect/troubleshooting.md.  Is that the right file?

Reply:
I don't think so. I can't tell what's going on here, the link they provided == this doc's path.